recipes/minvar/licence.md

```diff
@@ -0,0 +1,146 @@
+SOFTWARE LICENSE AGREEMENT (hereinafter referred to as AGREEMENT)
+=================================================================
+
+This is a legal agreement between you (hereinafter referred to as the LICENSEE)
+and the University of Zurich, Rämistrasse 71, 8006 Zurich (hereinafter referred
+to as UNIVERSITY OF ZURICH) pertaining to the right to use the software product
+“MinVar” (hereinafter referred to as SOFTWARE). Using the SOFTWARE indicates
+your acceptance of these terms and conclusion of the Agreement between the
```

Comment:
Right. Licence is custom and there is a licence file field, I thought to fill it but then I forgot. Is it ok if I remove `licence.md` and delete the `licence_file` field?